backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/audit/data/repository/DataExportAuditLogRepository.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.audit.data.repository;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import de.metas.audit.data.Action;
+import de.metas.audit.data.ExternalSystemParentConfigId;
+import de.metas.audit.data.model.DataExportAuditId;
+import de.metas.audit.data.model.DataExportAuditLog;
+import de.metas.audit.data.model.DataExportAuditLogId;
+import de.metas.process.PInstanceId;
+import de.metas.util.NumberUtils;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.compiere.model.I_Data_Export_Audit_Log;
+import org.springframework.stereotype.Repository;
+
+import java.util.Objects;
+
+import static org.adempiere.model.InterfaceWrapperHelper.saveRecord;
+
+@Repository
+public class DataExportAuditLogRepository
+{
+	private final IQueryBL queryBL = Services.get(IQueryBL.class);
+
+	@NonNull
+	public DataExportAuditLog save(@NonNull final DataExportAuditLog logEntry)
```

Comment:
better changed it to `DataExportAuditLog createNew(CreateDataExportAuditRequest request)`
(see my comment above)